src/1.1/index.ts

```diff
@@ -0,0 +1,7 @@
+process.stdin.setEncoding('utf-8')
+process.stdin.on('data', (data) => {
+    const incoming = data.toString().trim()
+    if (incoming) {
+        process.stdout.write(incoming.split('').reverse().join(''))
```

Comment:
[minor]
Not a functional advice, but it would be better to add line break for readability in console

`process.stdout.write(incoming.split('').reverse().join('') + '\n')`

So after reversing the string it would propose to enter input from a new line

Reply:
Nodemon restarts application after writing is finished and after new launch we have cursor on new line

Reply:
Tested on Windows. Application is not restared till Ctrl+C called.

Reply:
Yep, you was right, the line break is nice idea. Restarting of application was my phantom memory of this task from previous run.